tests/unit/lago/test_subnet_lease.py

```diff
@@ -117,3 +128,42 @@ def test_take_stale_lease(self, subnet_store, prefixes):
         network = subnet_store.acquire(prefixes[0].uuid_path)
         prefixes[0].remove()
         subnet_store.acquire(prefixes[1].uuid_path, str(network))
+
+    @pytest.mark.parametrize('num_prefixes', [7, 10, 55])
+    @pytest.mark.parametrize('remains', [0, 3, 6])
+    def test_release_several_prefixes(
+        self, subnet_store, prefix_mock_gen, num_prefixes, remains
+    ):
+        gen = prefix_mock_gen()
+        for _ in range(num_prefixes):
+            subnet_store.acquire(next(gen).uuid_path)
+        acquired = subnet_store.list_leases()
+        assert all(lease.valid for lease in acquired)
+        assert len(subnet_store.list_leases()) == num_prefixes
```

Comment:
Thanks, missed that.
About the triple '==' comparison, it might make the code more compact, but not pytest's output :)
When it fails, you'd see something like 3 numbers with inequalities, and it is hard to figure out what failed, so I prefer keeping each assert with one comparison, so when it fails - it is clear what went wrong.